packages/head/index.tsx

```diff
@@ -3,6 +3,7 @@ import Head from 'next/head'
 export default function HashiHead(props: HashiHeadProps): React.ReactElement {
   return (
     <Head>
+      {whenString(props.lang, <html lang={props.lang} />)}
```

Comment:
Just a quick check-in, does this work? the `html` tag would wrap `head`, so I would be surprised if this could mutate that tag.

Reply:
@BRKalow It does work, but I have been sitting on whether or not this would be the right solution vs getting the `i18n` next.config setting to work with cloud. Was getting an error on deployments with `next export`.